front/app/modules/commercial/project_folders/admin/containers/projects/ItemsInFolder.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { useState } from 'react';
```

Comment:
Extracted from the project folder settings (Folder projects tab). The same for the items (projects) not in the folder (see below).